### ISSUE TITLE
[Enhancement] Notify - define action as non-dismissing by prop

### ISSUE THREAD

**Issue by danikane** [quasarframework/quasar#1847]
Hi, I just saw you're adding an onDismiss handler to Notify, good stuff!
I was wondering whether you could add another prop to the config object for the action items, for example **willDismiss**, type _Boolean_, default _true_.
Currently the code for Notify is like:

```
        if (notif.actions) {
          notif.actions = config.actions.map(item => {
            const
              handler = item.handler,
              action = clone(item)
            action.handler = typeof handler === 'function'
              ? () => {
                handler()
                close()
              }
              : () => close()
            return action
          })
        }
```
Currently any action closes the Notify.
What I suggest is something like:

```
        if (notif.actions) {
          notif.actions = config.actions.map(item => {
            const
              willDismiss = item.willDismiss || true
              handler = item.handler,
              action = clone(item)
            action.handler = typeof handler === 'function'
              ? () => {
                handler()
                if (willDismiss) close()
              }
              : () => close()
            return action
          })
        }
```

Turn it around, make it a **noDismiss**, _true_ and for example
`
                noDismiss ? null : close()
`
if you think is better approach. :)
I can do a PR if this addition makes sense.

**Comment by pdanpdan** [comment]
Which would be the use case for a notification you want to keep visible after selecting an action?
The action should act as an acknowledge for the notification.
I think you can show another notification after the current one closes, because I would expect the action from the first one changes something.

**Comment by danikane** [comment]
Hi and thanks for asking. :)
The reason behind this request is a use case on a web map app. See screenshot below.
The Notify component is a great upgrade to the Toast and is used as a placeholder for info regarding a polygon drawn (therefore the 0 timeout). Whenever I dismiss the Notify I remove the polygon from the map.
The additional non-dismissing action would enable me for example to highlight/flash/pan to the shape drawn or all of the above from the same spot (the Notify component) without removing the shape.
 
![image](https://user-images.githubusercontent.com/18037666/38057926-b74bf484-32e9-11e8-866a-fdb2d55113c4.png)

**Comment by rstoenescu** [comment]
Added "noDismiss" prop to actions. Available in 0.15.11.

**Comment by danikane** [comment]
Amazing! Thanks, man. 😄
